internal/services/install.go

```
@@ -0,0 +1,275 @@
+		Host:    config.Host,
+		User:    config.User,
+		Port:    config.Port,
+		KeyPath: config.KeyPath,
+		Timeout: config.Timeout,
+	}
+
+	// Connect to remote host
+	if err := s.client.Connect(ctx, sshConfig); err != nil {
+		return fmt.Errorf("failed to connect to %s: %w", config.Target, err)
+	}
+	defer s.client.Disconnect()
```

golangci-lint (1.64.8)

Error return value of `s.client.Disconnect` is not checked

(errcheck)